crates/nargo_cli/src/main.rs

```diff
@@ -7,9 +7,11 @@ const PANIC_MESSAGE: &str = "This is a bug. We may have already fixed this in ne
 
 fn main() -> eyre::Result<()> {
     // Register a panic hook to display more readable panic messages to end-users
-    let (panic_hook, _) =
-        HookBuilder::default().display_env_section(false).panic_section(PANIC_MESSAGE).into_hooks();
-    panic_hook.install();
+    HookBuilder::default()
+    .display_env_section(false)
+    .display_location_section(cfg!(debug_assertions))
```

Comment:
```suggestion
    .display_location_section(false)
```
After experimenting with just always disabling this locally, I think we can go through with just never showing the Location portion. Actual panics will still print the stack trace so debugging is unaffected.